### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.1.2

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
+
+<!-- 8.1.2 START -->
+
+# 8.1.2 (2021-08-19)
+
+### Features and enhancements
+
+* **AzureMonitor:** Support PostgreSQL and MySQL Flexible Servers. [#38075](https://github.com/grafana/grafana/pull/38075), [@joshhunt](https://github.com/joshhunt)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:8]:
```suggestion
* **AzureMonitor:** Add support PostgreSQL and MySQL Flexible Servers. [#38075](https://github.com/grafana/grafana/pull/38075), [@joshhunt](https://github.com/joshhunt)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+* **Explore:** Add span duration to left panel in trace viewer. [#37806](https://github.com/grafana/grafana/pull/37806), [@connorlindsey](https://github.com/connorlindsey)
+* **Plugins:** Use file extension allowlist when serving plugin assets instead of checking for UNIX executable. [#37688](https://github.com/grafana/grafana/pull/37688), [@wbrowne](https://github.com/wbrowne)
+* **Profiling:** Support binding pprof server to custom network interfaces. [#36580](https://github.com/grafana/grafana/pull/36580), [@cinaglia](https://github.com/cinaglia)
+* **Search:** Make search icon keyboard navigable. [#37865](https://github.com/grafana/grafana/pull/37865), [@tskarhed](https://github.com/tskarhed)
+* **Template variables:** Keyboard navigation improvements. [#38001](https://github.com/grafana/grafana/pull/38001), [@tskarhed](https://github.com/tskarhed)
+* **Tooltip:** display ms within minute time range. [#37569](https://github.com/grafana/grafana/pull/37569), [@nikki-kiga](https://github.com/nikki-kiga)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:15]:
```suggestion
* **Tooltip:** Display ms within minute time range. [#37569](https://github.com/grafana/grafana/pull/37569), [@nikki-kiga](https://github.com/nikki-kiga)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+
+# 8.1.2 (2021-08-19)
+
+### Features and enhancements
+
+* **AzureMonitor:** Support PostgreSQL and MySQL Flexible Servers. [#38075](https://github.com/grafana/grafana/pull/38075), [@joshhunt](https://github.com/joshhunt)
+* **Datasource:** Change HTTP status code for failed datasource health check to 400. [#37895](https://github.com/grafana/grafana/pull/37895), [@stephaniehingtgen](https://github.com/stephaniehingtgen)
+* **Explore:** Add span duration to left panel in trace viewer. [#37806](https://github.com/grafana/grafana/pull/37806), [@connorlindsey](https://github.com/connorlindsey)
+* **Plugins:** Use file extension allowlist when serving plugin assets instead of checking for UNIX executable. [#37688](https://github.com/grafana/grafana/pull/37688), [@wbrowne](https://github.com/wbrowne)
+* **Profiling:** Support binding pprof server to custom network interfaces. [#36580](https://github.com/grafana/grafana/pull/36580), [@cinaglia](https://github.com/cinaglia)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:12]:
```suggestion
* **Profiling:** Add support for binding pprof server to custom network interfaces. [#36580](https://github.com/grafana/grafana/pull/36580), [@cinaglia](https://github.com/cinaglia)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+* **Search:** Make search icon keyboard navigable. [#37865](https://github.com/grafana/grafana/pull/37865), [@tskarhed](https://github.com/tskarhed)
+* **Template variables:** Keyboard navigation improvements. [#38001](https://github.com/grafana/grafana/pull/38001), [@tskarhed](https://github.com/tskarhed)
+* **Tooltip:** display ms within minute time range. [#37569](https://github.com/grafana/grafana/pull/37569), [@nikki-kiga](https://github.com/nikki-kiga)
+
+### Bug fixes
+
+* **Alerting:** fix saving LINE contact point. [#37744](https://github.com/grafana/grafana/pull/37744), [@xy-man](https://github.com/xy-man)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:19]:
```suggestion
* **Alerting:** Fix saving LINE contact point. [#37744](https://github.com/grafana/grafana/pull/37744), [@xy-man](https://github.com/xy-man)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,39 @@
...
+* **Alerting:** fix saving LINE contact point. [#37718](https://github.com/grafana/grafana/pull/37718), [@xy-man](https://github.com/xy-man)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:20]:
```suggestion
* **Alerting:** Fix saving LINE contact point. [#37718](https://github.com/grafana/grafana/pull/37718), [@xy-man](https://github.com/xy-man)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,39 @@
...
+* **Annotations:** Fix alerting annotation coloring. [#37412](https://github.com/grafana/grafana/pull/37412), [@kylebrandt](https://github.com/kylebrandt)
+* **Annotations:** Fixes so alert annotations are visible in the correct Panel. [#37959](https://github.com/grafana/grafana/pull/37959), [@hugohaggmark](https://github.com/hugohaggmark)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:22]:
```suggestion
* **Annotations:** Alert annotations are now visible in the correct Panel. [#37959](https://github.com/grafana/grafana/pull/37959), [@hugohaggmark](https://github.com/hugohaggmark)
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review, changes should be pushed upstream.

File: CHANGELOG.md (hunk 1 above)
Comment (joshhunt) [reply CHANGELOG.md:8]: Add support _for_ PostgreSQL...

Comment (FZambia) [reply CHANGELOG.md:8]: Fixed right inside a suggestion

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,39 @@
...
+* **Dashboard:** Prevent incorrect panel layout by comparing window width against theme breakpoints. [#37868](https://github.com/grafana/grafana/pull/37868), [@ashharrison90](https://github.com/ashharrison90)
+* **Elasticsearch:** Fix metric names for alert queries. [#37871](https://github.com/grafana/grafana/pull/37871), [@dsotirakis](https://github.com/dsotirakis)
+* **Explore:** Fix showing of full log context. [#37442](https://github.com/grafana/grafana/pull/37442), [@ivanahuckova](https://github.com/ivanahuckova)
+* **PanelEdit:** Fix 'Actual' size by passing the correct panel size to Das…. [#37885](https://github.com/grafana/grafana/pull/37885), [@ashharrison90](https://github.com/ashharrison90)
+* **Plugins:** Fix TLS datasource settings. [#37797](https://github.com/grafana/grafana/pull/37797), [@wbrowne](https://github.com/wbrowne)
+* **Variables:** Fix issue with empty drop downs on navigation. [#37776](https://github.com/grafana/grafana/pull/37776), [@hugohaggmark](https://github.com/hugohaggmark)
+* **Variables:** Fix url util converting `false` into `true`. [#37402](https://github.com/grafana/grafana/pull/37402), [@simPod](https://github.com/simPod)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:30]:
```suggestion
* **Variables:** Fix URL util converting `false` into `true`. [#37402](https://github.com/grafana/grafana/pull/37402), [@simPod](https://github.com/simPod)
```

**Review by achatterjee-grafana — APPROVED** [review]
One more change, otherwise LGTM

File: docs/sources/release-notes/release-notes-8-1-2.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+* **Plugins:** Use file extension allowlist when serving plugin assets instead of checking for UNIX executable. [#37688](https://github.com/grafana/grafana/pull/37688), [@wbrowne](https://github.com/wbrowne)
+* **Profiling:** Support binding pprof server to custom network interfaces. [#36580](https://github.com/grafana/grafana/pull/36580), [@cinaglia](https://github.com/cinaglia)
+* **Search:** Make search icon keyboard navigable. [#37865](https://github.com/grafana/grafana/pull/37865), [@tskarhed](https://github.com/tskarhed)
+* **Template variables:** Keyboard navigation improvements. [#38001](https://github.com/grafana/grafana/pull/38001), [@tskarhed](https://github.com/tskarhed)
+* **Tooltip:** display ms within minute time range. [#37569](https://github.com/grafana/grafana/pull/37569), [@nikki-kiga](https://github.com/nikki-kiga)
+
+### Bug fixes
+
+* **Alerting:** fix saving LINE contact point. [#37744](https://github.com/grafana/grafana/pull/37744), [@xy-man](https://github.com/xy-man)
+* **Alerting:** fix saving LINE contact point. [#37718](https://github.com/grafana/grafana/pull/37718), [@xy-man](https://github.com/xy-man)
```
Comment (achatterjee-grafana) [inline docs/sources/release-notes/release-notes-8-1-2.md:25]: Change "fix" to "Fix".

File: docs/sources/release-notes/release-notes-8-1-2.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+* **Auth:** Hide SigV4 config UI and disable middleware when its config flag is disabled. [#37293](https://github.com/grafana/grafana/pull/37293), [@wbrowne](https://github.com/wbrowne)
+* **Dashboard:** Prevent incorrect panel layout by comparing window width against theme breakpoints. [#37868](https://github.com/grafana/grafana/pull/37868), [@ashharrison90](https://github.com/ashharrison90)
+* **Elasticsearch:** Fix metric names for alert queries. [#37871](https://github.com/grafana/grafana/pull/37871), [@dsotirakis](https://github.com/dsotirakis)
+* **Explore:** Fix showing of full log context. [#37442](https://github.com/grafana/grafana/pull/37442), [@ivanahuckova](https://github.com/ivanahuckova)
+* **PanelEdit:** Fix 'Actual' size by passing the correct panel size to Das…. [#37885](https://github.com/grafana/grafana/pull/37885), [@ashharrison90](https://github.com/ashharrison90)
+* **Plugins:** Fix TLS datasource settings. [#37797](https://github.com/grafana/grafana/pull/37797), [@wbrowne](https://github.com/wbrowne)
```
Comment (achatterjee-grafana) [inline docs/sources/release-notes/release-notes-8-1-2.md:33]:
```suggestion
* **Plugins:** Fix TLS data source settings. [#37797](https://github.com/grafana/grafana/pull/37797), [@wbrowne](https://github.com/wbrowne)
```

File: docs/sources/release-notes/release-notes-8-1-2.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+* **Variables:** Fix issue with empty drop downs on navigation. [#37776](https://github.com/grafana/grafana/pull/37776), [@hugohaggmark](https://github.com/hugohaggmark)
+* **Variables:** Fix url util converting `false` into `true`. [#37402](https://github.com/grafana/grafana/pull/37402), [@simPod](https://github.com/simPod)
```
Comment (achatterjee-grafana) [inline docs/sources/release-notes/release-notes-8-1-2.md:35]:
```suggestion
* **Variables:** Fix URL util converting `false` into `true`. [#37402](https://github.com/grafana/grafana/pull/37402), [@simPod](https://github.com/simPod)
```

File: docs/sources/release-notes/release-notes-8-1-2.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+* **Explore:** Add span duration to left panel in trace viewer. [#37806](https://github.com/grafana/grafana/pull/37806), [@connorlindsey](https://github.com/connorlindsey)
+* **Plugins:** Use file extension allowlist when serving plugin assets instead of checking for UNIX executable. [#37688](https://github.com/grafana/grafana/pull/37688), [@wbrowne](https://github.com/wbrowne)
+* **Profiling:** Support binding pprof server to custom network interfaces. [#36580](https://github.com/grafana/grafana/pull/36580), [@cinaglia](https://github.com/cinaglia)
+* **Search:** Make search icon keyboard navigable. [#37865](https://github.com/grafana/grafana/pull/37865), [@tskarhed](https://github.com/tskarhed)
+* **Template variables:** Keyboard navigation improvements. [#38001](https://github.com/grafana/grafana/pull/38001), [@tskarhed](https://github.com/tskarhed)
+* **Tooltip:** display ms within minute time range. [#37569](https://github.com/grafana/grafana/pull/37569), [@nikki-kiga](https://github.com/nikki-kiga)
```
Comment (achatterjee-grafana) [inline docs/sources/release-notes/release-notes-8-1-2.md:20]:
```suggestion
* **Tooltip:** Display milliseconds (ms) within minute time range. [#37569](https://github.com/grafana/grafana/pull/37569), [@nikki-kiga](https://github.com/nikki-kiga)
```

File: docs/sources/release-notes/release-notes-8-1-2.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+list = false
++++
+
+<!-- Auto generated by update changelog github action -->
+
+# Release notes for Grafana 8.1.2
+
+### Features and enhancements
+
+* **AzureMonitor:** Support PostgreSQL and MySQL Flexible Servers. [#38075](https://github.com/grafana/grafana/pull/38075), [@joshhunt](https://github.com/joshhunt)
```
Comment (achatterjee-grafana) [inline docs/sources/release-notes/release-notes-8-1-2.md:13]:
```suggestion
* **AzureMonitor:** Add support for PostgreSQL and MySQL Flexible Servers. [#38075](https://github.com/grafana/grafana/pull/38075), [@joshhunt](https://github.com/joshhunt)
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,39 @@
...
+* **Auth:** Hide SigV4 config UI and disable middleware when its config flag is disabled. [#37293](https://github.com/grafana/grafana/pull/37293), [@wbrowne](https://github.com/wbrowne)
+* **Dashboard:** Prevent incorrect panel layout by comparing window width against theme breakpoints. [#37868](https://github.com/grafana/grafana/pull/37868), [@ashharrison90](https://github.com/ashharrison90)
+* **Elasticsearch:** Fix metric names for alert queries. [#37871](https://github.com/grafana/grafana/pull/37871), [@dsotirakis](https://github.com/dsotirakis)
+* **Explore:** Fix showing of full log context. [#37442](https://github.com/grafana/grafana/pull/37442), [@ivanahuckova](https://github.com/ivanahuckova)
+* **PanelEdit:** Fix 'Actual' size by passing the correct panel size to Das…. [#37885](https://github.com/grafana/grafana/pull/37885), [@ashharrison90](https://github.com/ashharrison90)
+* **Plugins:** Fix TLS datasource settings. [#37797](https://github.com/grafana/grafana/pull/37797), [@wbrowne](https://github.com/wbrowne)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:28]:
```suggestion
* **Plugins:** Fix TLS data source settings. [#37797](https://github.com/grafana/grafana/pull/37797), [@wbrowne](https://github.com/wbrowne)
```

**Review by achatterjee-grafana — COMMENTED** [review]
Changes need to be made to changelogs and to release notes. Thanks!

File: docs/sources/release-notes/release-notes-8-1-2.md [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+* **Explore:** Add span duration to left panel in trace viewer. [#37806](https://github.com/grafana/grafana/pull/37806), [@connorlindsey](https://github.com/connorlindsey)
+* **Plugins:** Use file extension allowlist when serving plugin assets instead of checking for UNIX executable. [#37688](https://github.com/grafana/grafana/pull/37688), [@wbrowne](https://github.com/wbrowne)
+* **Profiling:** Support binding pprof server to custom network interfaces. [#36580](https://github.com/grafana/grafana/pull/36580), [@cinaglia](https://github.com/cinaglia)
+* **Search:** Make search icon keyboard navigable. [#37865](https://github.com/grafana/grafana/pull/37865), [@tskarhed](https://github.com/tskarhed)
+* **Template variables:** Keyboard navigation improvements. [#38001](https://github.com/grafana/grafana/pull/38001), [@tskarhed](https://github.com/tskarhed)
+* **Tooltip:** display ms within minute time range. [#37569](https://github.com/grafana/grafana/pull/37569), [@nikki-kiga](https://github.com/nikki-kiga)
+
+### Bug fixes
+
+* **Alerting:** fix saving LINE contact point. [#37744](https://github.com/grafana/grafana/pull/37744), [@xy-man](https://github.com/xy-man)
```
Comment (FZambia) [inline docs/sources/release-notes/release-notes-8-1-2.md:24]:
```suggestion
* **Alerting:** Fix saving LINE contact point. [#37744](https://github.com/grafana/grafana/pull/37744), [@xy-man](https://github.com/xy-man)
```

File: docs/sources/release-notes/release-notes-8-1-2.md (hunk 8 above)
Comment (FZambia) [inline docs/sources/release-notes/release-notes-8-1-2.md:25]:
```suggestion
* **Alerting:** Fix saving LINE contact point. [#37718](https://github.com/grafana/grafana/pull/37718), [@xy-man](https://github.com/xy-man)
```

**Comment by hugohaggmark** [comment]
@achatterjee-grafana @FZambia all changes need to be done in the PRs, not in the change log. The change log is automatically generated, so these changes will all go away when it's regenerated. If the release docs don't mention this, it might be worth writing this down. Thank you.

**Pull request merged**